projects/go-lib/src/lib/components/go-select/go-select.component.scss

```diff
@@ -0,0 +1,33 @@
+@import '~@tangoe/gosheets/base/_variables';
+@import '~@tangoe/gosheets/base/_mixins';
+@import "~@ng-select/ng-select/themes/default.theme.css";
+
+.ng-select .ng-select-container {
+  height: 1.5rem + $column-gutter !important;
```

Comment:
Oh man, what's up with this `!important`?

Reply:
Specificity issues from the ng-select...